src/components/paragraph/button/index.tsx

```diff
@@ -0,0 +1,29 @@
+import { v4 as uuidv4 } from 'uuid'
+import Link from 'next/link'
+
+function isRequestValid(paragraph: any) {
+  return (
+    paragraph.field_button_label !== null &&
+    paragraph.field_button_link !== null &&
+    paragraph.field_button_link?.uri !== null
+  )
+}
+
+const Button = ({ paragraph }): JSX.Element => {
+  if (!paragraph || !isRequestValid(paragraph)) return
+
+  const {
+    field_button_label: fieldButtonLabel,
+    field_button_link: fieldButtonLink,
+  } = paragraph
+
+  return (
+    <div key={uuidv4()}>
```

Comment:
`paragraph.id` exists and would also work here (or add it to the destructuring). Any Drupal data structure passed in will have its own uuid at `*.id`.  It might make sense to just use that.

Reply:
Done